source/scripts/Background/migration/v4_1_0.ts

```diff
@@ -0,0 +1,66 @@
+import { reload } from 'utils/browser';
+import IAssetListState from 'state/assets/types';
+import IContactBookState from 'state/contacts/types';
+import IPriceState from 'state/price/types';
+import IVaultState from 'state/vault/types';
+import ISwapState from 'state/swap/types';
+import IProvidersState from 'state/providers/types';
+import { INFTListState } from 'state/nfts/types';
+import { saveState } from 'state/localStorage';
+import { IBiometricsState } from 'state/biometrics/types';
+import { IDAppState } from 'state/dapp/types';
+
+type V4_0_4_State = {
+  assets: IAssetListState;
+  biometrics: IBiometricsState;
+  contacts: IContactBookState;
+  dapp: IDAppState;
+  nfts: INFTListState;
+  price: IPriceState;
+  providers: IProvidersState;
+  swap: ISwapState;
+  vault: IVaultState;
+};
+
+const updateAssets = (assets: IAssetListState, assetKey: string, properties: object) => {
+  const assetInfo = assets[assetKey];
+  if (!assetInfo) return assets;
+
+  const assetWithProperties = {
+    ...assetInfo,
+    ...properties,
+  };
+
+  const assetsUpdated = {
+    ...assets,
+    [assetKey]: assetWithProperties,
+  };
+
+  return assetsUpdated;
+};
+
+const MigrateRunner = async (oldState: any) => {
+  const DOR_ASSET_KEY = 'DAG0CyySf35ftDQDQBnd1bdQ9aPyUdacMghpnCuM-main2';
```

Comment:
What's `-main2`?

Reply:
That's the ID of the network -> Mainnet 2. We introduced this pattern because on EVM networks we could have tokens with the same contract address in different networks.

Reply:
Nice, oko